src/test/java/com/kodilla/ecommercee/ItemTestSuite.java

```diff
@@ -0,0 +1,168 @@
+package com.kodilla.ecommercee;
+
+import com.kodilla.ecommercee.domain.*;
+import com.kodilla.ecommercee.repository.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest()
```

Comment:
```suggestion
@SpringBootTest
```
taka drobnostka, ten nawias chyba można usunąć :)

Reply:
Owszem dzięki :)